playwright.config.js

```diff
@@ -61,14 +61,15 @@ export default defineConfig({
   /* Configure projects for major browsers */
   projects: [
     {
-      name: "chromium",
-      use: { ...devices["Desktop Chrome"] },
+      name: "infrastructure",
+      use: {
+        ...devices["Desktop Chrome"],
+        account: "infrastructure-committee.near",
+      },
     },
-    { name: "infrastructure", use: { ...devices["Desktop Chrome"] } },
-
     {
       name: "events",
-      testMatch: /events\/.*.spec.js/, // (proposal|events)
+      testMatch: /(events|proposal)\/.*.spec.js/, // (proposal|events)
```

Comment:
@petersalomonsen this change is not in the CI only locally. These tests don't work for the `events-commitee.near` account, that is why I added `events/proposals.spec.js`.

Reply:
good point, I did separate out the proposal comment test and added a workflow task for that as you can see here: https://github.com/NEAR-DevHub/neardevhub-bos/blob/main/.github/workflows/continuous-integration-workflow-events.yml#L22, but I forgot to set it for the whole proposal directory which I believe also will work. Will do so in the next PR.